README.md

```diff
@@ -13,25 +13,25 @@ dns.providers.duckdns
 
 To use this module for the ACME DNS challenge, [configure the ACME issuer in your Caddy JSON](https://caddyserver.com/docs/json/apps/tls/automation/policies/issuer/acme/) like so:
 
-```
+```json
```

Comment:
This is the only change I agree with.

I don't agree with tabs -> spaces. And you also did it inconsistently, some of the lines are still tabs.

![image](https://user-images.githubusercontent.com/2111701/103025152-31fe8a80-451f-11eb-82d6-7e0d847343c3.png)

So... no thanks.